framework/Models/ROM.py

```diff
@@ -1428,6 +1433,19 @@ def provideExpectedMetaKeys(self):
     metaParams.update(params)
     return metaKeys, metaParams
 
+  def copyModel(self, obj):
+    """
+      This method is aimed to copy the "obj" model in this instance
+      It is generally used for unpickling objects (models)
+      @ In, obj, instance, the instance of the object to copy from
+      @ Out, None
+    """
+    # save reseeding parameters from pickledROM
+    loadSettings = self.initializationOptionDict
```

Comment:
@wangcj05 this will cause problems with the new ROM API, possibly; can you check what issues this might raise?